src/python/competition.py

```diff
@@ -116,7 +116,13 @@ def __init__(self, checkout, dataSource,
                name = None,
                indexSort = None,
                vectorFile = None,
-               vectorDimension = None
+               vectorDimension = None,
+               # an int defining how many segments to rearrange to, 0 represents not performing rearrange
+               # rearrange /100 gives how many large segments desired
+               # (rearrange % 100) / 10 gives how many medium segments desired
+               # rearrange % 10 gives how many small segments desired
+               # For example, rearrange = 555 means 5 large segments, 5 medium segments and 5 small segments
```

Comment:
Aha!  This is useful documentation!  Maybe we can get this into `Indexer` help output?

Reply:
Hmmm? I can't find it? Do we have a help for `Indexer`? I thought that was intended to be called only by python scripts?

Reply:
LOL maybe it does not have any help at all!  It's fine, we can fix this later!